From 917d3565b5c6f064250fece1eebd7dc0f78a1fc0 Mon Sep 17 00:00:00 2001
From: Bin Meng <bmeng.cn@gmail.com>
Date: Sat, 7 Oct 2017 02:43:52 -0700
Subject: [PATCH 006/105] x86: minnowmax: Adjust VGA rom address

Adjust VGA rom address to 0xfffb0000 so that u-boot.rom image
can be built again.

Signed-off-by: Bin Meng <bmeng.cn@gmail.com>
Reviewed-by: Stefan Roese <sr@denx.de>
---
 configs/minnowmax_defconfig | 1 +
 doc/README.x86              | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/minnowmax_defconfig b/configs/minnowmax_defconfig
index fc2564cd7d..d1332d369d 100644
--- a/configs/minnowmax_defconfig
+++ b/configs/minnowmax_defconfig
@@ -6,6 +6,7 @@ CONFIG_INTERNAL_UART=y
 CONFIG_DEBUG_UART=y
 CONFIG_SMP=y
 CONFIG_HAVE_VGA_BIOS=y
+CONFIG_VGA_BIOS_ADDR=0xfffb0000
 CONFIG_GENERATE_PIRQ_TABLE=y
 CONFIG_GENERATE_MP_TABLE=y
 CONFIG_GENERATE_ACPI_TABLE=y
diff --git a/doc/README.x86 b/doc/README.x86
index c96a22cb08..92f41283be 100644
--- a/doc/README.x86
+++ b/doc/README.x86
@@ -320,7 +320,7 @@ Offset   Description         Controlling config
 6ef000   Environment         CONFIG_ENV_OFFSET
 6f0000   MRC cache           CONFIG_ENABLE_MRC_CACHE
 700000   u-boot-dtb.bin      CONFIG_SYS_TEXT_BASE
-790000   vga.bin             CONFIG_VGA_BIOS_ADDR
+7b0000   vga.bin             CONFIG_VGA_BIOS_ADDR
 7c0000   fsp.bin             CONFIG_FSP_ADDR
 7f8000   <spare>             (depends on size of fsp.bin)
 7ff800   U-Boot 16-bit boot  CONFIG_SYS_X86_START16
-- 
2.13.6

